internal/command/root.go

```diff
@@ -184,9 +178,9 @@ func runExec(options execOptions) error {
 	}
 
 	errCh := make(chan error, 1)
+	defer close(errCh)
```

Comment:
Here is a bug fix, if `errCh` is closed in the go channel we can't handle it at line194.